docs/guide/querying.rst

```diff
@@ -218,12 +218,36 @@ However, this doesn't map well to the syntax so you can also use a capital S ins
 
 Raw queries
 -----------
-It is possible to provide a raw :mod:`PyMongo` query as a query parameter, which will
-be integrated directly into the query. This is done using the ``__raw__``
+It is possible to provide a raw :mod:`PyMongo` query as a query parameter or update as a update parameter , which will
+be integrated directly into the query or update. This is done using the ``__raw__``
 keyword argument::
 
     Page.objects(__raw__={'tags': 'coding'})
 
+    # or for update
+
+    Page.objects(__raw__={'tags': 'coding'}).update(__raw__={'$set': {'tags': 'coding'}})
+
+    Page.objects(tags='coding').update(__raw__={'$set': {'tags': 'coding'}})
+
+.. versionadded:: 0.4
+
+
+Update with Aggregation Pipeline
+-----------
+It is possible to provide a raw :mod:`PyMongo` aggregation update parameter, which will
+be integrated directly into the update. This is done using the ``aggregation_update=True`` and ``__raw__``
+pipeline
+`Update with Aggregation Pipeline <https://docs.mongodb.com/manual/reference/method/db.collection.updateMany/#update-with-aggregation->`_
+keyword argument::
+
+    # 'tags' field is set to 'coding is fun'
+    Page.objects(tags='coding').update(__raw__=[
+        {"$set": {"tags": {"$concat": ["$tags", "is fun"]}}}
+        ],
+        aggregation_update=True,
```

Comment:
To be honest, I'm all in for adding support for update aggregation pipeline but I'm not sure about this interface, I'm talking here about `aggregation_update=True` which is not really needed as we could understand that from the fact that a list/tuple is provided in __raw__

Pymongo's native interface simply takes a dict for regular updates or a list for aggregation pipeline, so this would be closer to that somehow.

What do you think?

Reply:
Great point 
Just update the PR